enterprise/views.py

```diff
@@ -294,3 +297,59 @@ def post(self, request):
             # We're handing consent only for a particular course
             return self.post_course_specific_consent(request, specific_course, consent_provided)
         return self.post_account_consent(request, consent_provided)
+
+
+class RouteLearnerData(APIView):
+    """
+    Provide an endpoint to route learner data to the appropriate channels based on Enterprise's configuration.
+    """
+
+    def get(self, request):
+        """
+        It is unclear if a get request should do anything at this point.
+        """
+        return Response(request.data)
+
+    def post(self, request):
+        """
+        Hit when certificates are generated or when audit learners in self paced courses reach a passing grade.
+
+        When this happens, this method will route the learners data appropriately.
+
+        For a given learner who has completed a course, find out the following:
+            if the learner is associated with an enterprise customer.
+            if the course is part of an enterprise enrollment.
+            if the enterprise customer is configured for SuccessFactors.
+
+        Continue if all the conditions above are met, otherwise abort processing.
+
+        Based on the EnterpriseCustomerPluginConfiguration for the Enterprise, determine which endpoints are
+        configured to receive learner data, and send a formatted request to each of those endpoints.
+        If nothing is configured, abort processing.
+        Note that the request format will be uniform across the different plugin endpoints.
+        """
+        return Response(request.data)
```

Comment:
@brittneyexline It doesn't seem to do anything - just echo the request. Is it supposed to be so? If that's just a dummy implementation supposed to be overridden by nested classes, I'd suggest replacing it with `raise NotImplementedError("Must override in descendent class")` or something like that.